ui/src/views/Rule/RuleDetail.js

```diff
@@ -29,6 +29,8 @@ import HitCounterCard from "../Analytic/HitCounterCard";
 
 import axios from "axios";
 
+import localeJson from "../../jsondata/locale.json";
```

Comment:
I think it's better to access those static list of locale using a (utility) function. Not directly access the data.

Reply:
now using function getLocales from hotstone.js